src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -824,7 +824,8 @@
                 </Canvas.Background>
             </Canvas>
 
-            <controls:Watch3DView x:Name="BackgroundPreview" 
+            <controls:Watch3DView x:Name="BackgroundPreview"
+                                  x:FieldModifier="public"
```

Comment:
Again I needed to access this property for the `Watch3DView` and `Viewport3DX` objects.